test/api_integration/apis/saved_objects/bulk_create.ts

```diff
@@ -97,10 +97,11 @@ export default function ({ getService }: FtrProviderContext) {
       before(
         async () =>
           // just in case the kibana server has recreated it
-          await es.indices.delete({ index: '.kibana' }, { ignore: [404] })
+          await es.indices.delete({ index: '.kibana*' }, { ignore: [404] })
```

Comment:
I've introduced several wildcard delete operations which will need to be replaced by the `esDeleteAllIndices` service in https://github.com/elastic/kibana/pull/88986